packages/core/src/shared/utilities/collectionUtils.ts

```diff
@@ -329,6 +330,18 @@ export function partialClone(obj: any, depth: number = 3, omitKeys: string[] = [
     return clonedObj
 }
 
+/**
+ * Wrapper around nodes inspect function that works on web. Defaults to JSON.stringify on web.
+ * @param obj object to show
+ * @param opt options for showing (ex. depth, omitting keys)
+ */
+export function inspect(obj: any, opt?: InspectOptions): string {
```

Comment:
if we're only using "depth" then it's probably more confusing than helpful to reference `InspectOptions` here
```suggestion
export function inspect(obj: any, opt?: { depth: number }): string {
```